test/unit/model/import/CvsItemResultTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/*
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021  (original work) Open Assessment Technologies SA;
+ */
+
+namespace oat\taoQtiItem\model\import;
```

Comment:
We need to use the test namespaces like here `oat\taoQtiItem\test\unit\model\import\...`

https://github.com/oat-sa/extension-tao-itemqti/pull/1698/files#diff-176632c1dcaa779e3c865f234aece0c756730b38bc8f0c17729c536c3a001211R23